trainingMaquetado/Maquetado.css

```diff
@@ -0,0 +1,49 @@
+hr {
+	background-color: #F5C85F;
+	width: 4px;
+	height: 300px;
+	margin: 60px;
+	border: none;
+  border-radius: 60px;
+  flex-shrink: 0; 
+  background: radial-gradient(#F5C85F 40%, white 60%);
+}
+
+.headline {
```

Comment:
you don't need this here, you can add a `border-top: 4px solid #F5C85F` in the `.upper-container` to achieve this